chainer/functions/array/rollaxis.py

```diff
@@ -0,0 +1,60 @@
+from chainer import cuda
+from chainer import function
+from chainer.utils import type_check
+
+
+class Rollaxis(function.Function):
+
+    """Roll axis of an array."""
+
+    def __init__(self, axis, start):
+        self.axis = int(axis)
```

Comment:
We need to check if arguments are int. But, OK, i'll replace this code with a simple if statement.